tests/llama_index/test_llama_index_tracer.py

```diff
@@ -77,6 +77,15 @@ def test_trace_llm_complete(is_async):
             "prompt_tokens_details": None,
         }.items()
     )
+    assert (
+        attr[SpanAttributeKey.CHAT_USAGE].items()
+        >= {
+            TokenUsageKey.INPUT_TOKENS: 5,
+            TokenUsageKey.OUTPUT_TOKENS: 7,
+            TokenUsageKey.TOTAL_TOKENS: 12,
+        }.items()
+    )
```

Comment:
```suggestion
    assert attr[SpanAttributeKey.CHAT_USAGE] == {
        TokenUsageKey.INPUT_TOKENS: 5,
        TokenUsageKey.OUTPUT_TOKENS: 7,
        TokenUsageKey.TOTAL_TOKENS: 12,
    }
```

Let's use exact match